config/default.json

```diff
@@ -3,10 +3,18 @@
 	"blacklistedDomains": [],
 	"deproxifyURLs": false, // Automatically try deproxified versions of URLs
 	"identifierSearchLambda": "", // Identifier search Lambda function for text search
+	"recognizerLambda": "", // PDF recognizer Lambda function
 	"port": 1969,
 	"translators": {
 		"CrossrefREST.email": "" // Pass an email to Crossref REST API to utilize the faster servers pool
 	},
+	"s3Upload": {
+		"params": {
+			"Bucket": ""
+		},
+		"accessKeyId": "",
```

Comment:
We are just passing the whole `s3Upload` object as a S3 config. It's useful because there can be other configuration options that are needed to pass i.e. `region`. But the variable naming doesn't depend on us.